crypto/crypto.ts

```diff
@@ -204,92 +194,60 @@ const stdCrypto: StdCrypto = ((x) => x)({
       data: BufferSource | AsyncIterable<BufferSource> | Iterable<BufferSource>,
     ): Promise<ArrayBuffer> {
       const { name, length } = normalizeAlgorithm(algorithm);
-
-      assertValidDigestLength(length);
-
-      const bytes = bufferSourceBytes(data);
-
-      // We delegate to WebCrypto whenever possible,
-      if (
-        // if the algorithm is supported by the WebCrypto standard,
-        (WEB_CRYPTO_DIGEST_ALGORITHM_NAMES as readonly string[]).includes(
-          name,
-        ) &&
-        // and the data is a single buffer,
-        bytes
-      ) {
-        return webCrypto.subtle.digest(algorithm, bytes);
-      } else if (DIGEST_ALGORITHM_NAMES.includes(name as DigestAlgorithmName)) {
-        if (bytes) {
+      if (DIGEST_ALGORITHM_NAMES.includes(name as DigestAlgorithmName)) {
```

Comment:
This seems to change the priority of the crypto implementation to be used.

Currently `digest` uses native Web Crypto `subtle.digest` if the given algorithm is one of 4 standard algorithms, but now it uses `stdCrypto.subtle.digestSync` (therefore `wasmCrypto.digest`). Is this change intentional?

Reply:
Made some tweaks now.